advertools/sitemaps.py

```diff
@@ -412,10 +412,10 @@
 
 logging.basicConfig(level=logging.INFO)
 
-headers = {"User-Agent": "advertools-" + version}
+DEFAULT_HEADERS = {"User-Agent": "advertools-" + version}
```

Comment:
Just curious, why it is in all caps? If it's to comply with Python PEP8 on constants, you're absolutely right.

Because some people already use the trick `adv.sitemap.headers['User-agent']` to change the UA, this might break their code, so it's probably better to keep it in lower case. Otherwise please let me know.